internal/db/remote/commit/templates/dump_initial_migration.sh

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+# Explanation of special flags:
+#
+#   --schema-only     omit data like migration history, pgsodium key, etc.
+#   --exclude-schema  omit internal schemas as they are maintained by platform
+#   --no-comments     only object owner can set comment, omit to allow restore by non-superuser
+#   --extension '*'   prevents event triggers from being dumped, bash escaped with single quote
 pg_dump \
     --schema-only \
     --quote-all-identifier \
     --exclude-schema "$EXCLUDED_SCHEMAS" \
-    --schema '*' \
     --extension '*' \
+    --no-comments \
     --dbname "$DB_URL" \
-| sed 's/CREATE SCHEMA "public"/-- CREATE SCHEMA "public"/' \
```

Comment:
We shouldn't remove this - the `public` schema already exists.

Reply:
I just checked that `create schema "public"` statement is no longer dumped since the removal of `--schema '*'` flag.